docker-compose.yml

```diff
@@ -89,6 +89,8 @@ services:
       context: ./src
       dockerfile: worker/Dockerfile.vdb-upload-worker
     container_name: vectorflow_vdb_upload_worker
+    volumes:
+      - ./src/:/app
```

Comment:
@EteimZ  thanks for figuring this out!! Does this still enable running in detached mode or will we always have to have the terminal window that runs `docker-compose up` open for it to work/?

Reply:
just want to understand a bit more if it should be in the default workflow, and if it is, we should add it to all the containers where its relevant

Reply:
This method works even in detached mode. The only draw back is I haven't figured out how to properly set it up with my IDE. So currently I am not getting autocomplete and intellisense while developing with it. I'm using this [resource](https://code.visualstudio.com/docs/devcontainers/tutorial) to try to get autocomplete and intellisense set up on vscode.